### PR TITLE
Task2.3

### PR DESCRIPTION
**Pull request by Vomchick**

### CODE REVIEW

File: Task2.3/Classes/Trainig.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            {
+                if (Lessons[i] is not PracticalLesson)
+                {
+                    isPractical = false;
+                    break;
+                }
+            }
+            return isPractical;
+        }
+
```
Comment (IgorKheidorov) [inline Task2.3/Classes/Trainig.cs]: Think about creation (or use standard IClonable) and implementation of interface for  clone

File: Task2.3/Classes/Trainig.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
+﻿using System;
```
Comment (IgorKheidorov) [inline Task2.3/Classes/Trainig.cs]: remove

File: Task2.3/Classes/Trainig.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                if (Lessons[i] is not PracticalLesson)
+                {
+                    isPractical = false;
+                    break;
+                }
+            }
+            return isPractical;
+        }
+
+        public void Clone(Trainig trainingToCopy)
```
Comment (IgorKheidorov) [inline Task2.3/Classes/Trainig.cs]: Clone() has to create object to clone to itself, not takes an empty object as parameter-> no guarantee that this object will be empty

File: Task2.3/Classes/Trainig.cs [hunk 4]
```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2._3.Classes
+{
+    public class Trainig
+    {
```
Comment (IgorKheidorov) [inline Task2.3/Classes/Trainig.cs]: private field without any chances to set it?)))

File: Task2.3/Classes/Trainig.cs [hunk 5]
```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2._3.Classes
+{
```
Comment (IgorKheidorov) [inline Task2.3/Classes/Trainig.cs]: Add one common enity (class) for Lesson and Training (containing Description)

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Think about IClonable interface and common base class for Lesson and Training

File: Task2.3/Classes/Trainig.cs (hunk 4 above)
Comment (Vomchick) [reply Task2.3/Classes/Trainig.cs]: That supposed to be a property... but ctrl+c...

File: Task2.3/Classes/TrainigBase.cs [hunk 6]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
```
Comment (IgorKheidorov) [inline Task2.3/Classes/TrainigBase.cs]: remove

File: Task2.3/Classes/Trainig.cs [hunk 7]
```diff
@@ -0,0 +1,56 @@
+﻿namespace Task2._3.Classes;
+
```
Comment (IgorKheidorov) [inline Task2.3/Classes/Trainig.cs]: Training, spelling

**Review by IgorKheidorov — APPROVED** [review]
Fix comment and merge

**Pull request merged**